src/main/resources/config/application.properties

```diff
@@ -0,0 +1,41 @@
+info.service.name=shared-id-endpoint
+
+aws.region=us-east-1
+
+metrics.cloud-watch.enabled=true
+metrics.cloud-watch.namespace=shared-id-endpoint
+
+cookies.secure=false
+
+cookie.audit.name=audit
+cookie.audit.ttl=2419200
+cookie.audit.cipher-key=L83I1HNHIX16FXZ9
+cookie.shared-id.name=sharedid
+cookie.shared-id.ttl=2419200
+cookie.shared-id.opt-out-value=00000000000000000000000000
+cookie.shared-id.legacy-opt-out-value=00000000-0000-0000-0000-000000000000
```

Comment:
Do we really need legacy value? Not like the Magnite ID system, from which Shared Id cloned, Shared Id is a new system, no browser suppose to have legacy value for domain `id.sharedid.org`
If we can get rid of legacy value setting and code supporting it, it will benefit future maintenance.
(the same concern for cookie.opt-out.legacy)

Reply:
Get resolved.